monty_hall_function.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+from enum import Enum
+
+class Strategie(Enum):
+    CHANGER = 1
+    GARDER = 2
+    
+def Monty_Hall(nb_de_parties = 10000):
+    
+    '''Simule n parties du jeu de Monty Hall
+    
+    Cette fonction simule n = nb_de_parties du jeu
+    de Monty Hall pour chaque strategie "CHANGER" et
+    "GARDER". Elle retourne les gains de chaque
+    strategie
+    
+    args :
+        nb_de_parties (int) : Nombre de parties
+        de chaque strategie
+        
+    return : 
+        np.array (2,) : Gains de chaque methodes, 
+        d'abord "CHANGER" puis "GARDER"
+    '''
+    #Bonne porte au hasard entre 0 et 2 pour toutes les parties
+    bonne_porte = np.random.randint(0, 3, nb_de_parties)
+
+    #Premier choix au hasard entre 0 et 2 pour toutes les parties
+    premier_choix = np.random.randint(0, 3, nb_de_parties)
+    
+    #Quand on garde, deuxieme choix = premier choix logique
```

Comment:
No need to put as many comment, ideally the code should always speak for itself unless it is too complicated and therefore needs additional explanation in the comments.